### ISSUE TITLE
Peer Review - Document Organization and Template Structure

### ISSUE THREAD

**Issue by harrisonchiu** [RezaJodeiri/CXR-Capstone#46]
**Artifact Under Review**
Document Organization criteria in the rubric and section 4 Specific System Description

**Team Number for Team Doing the Review**
Team 16

**Description of Issue**
The SRS rubric for the Document Organization criteria says to "Follow template; if there are changes, they are clearly documented". In your section 4.2, many sections 4.2.3 onwards change the template's section and content. For example, 4.2.5 should be for "Theoretical Models" and contains many equations explained in depth, but your section is "Properties of a Correct Solution". These sections could be included to explain useful equations used in neural networks that are more in-depth than the general ones mentioned in your section 5.1. Equations could include optimizing the lower bound of a distribution (ELBO) or training the visual autoencoder.
![image](https://github.com/user-attachments/assets/d4a818cc-54b5-47f4-8756-4477a06ddcd4)
(Your equation may be different depending on how your model extracts features.)

Basically, there is a lot of content, equations (and its explanation) that can be included to match the SRS template.
